src/Applications/LDAS_App/ldas_setup

```diff
@@ -384,12 +392,22 @@ class LDASsetup:
                     landpertRstFile=tmpRstDir+'/'+tmpFile
                     if ( os.path.isfile(landpertRstFile)) :
                         self.has_geos_pert = True
-                    else :
-                        ldassa_tmp=self.rqdExeInp['RESTART_ID']+'.ens0000.pert_ldas_rst.'+y4m2d2_h2m2+'z.bin'
-                        LDASsa_pertRstFile=tmpRstDir+'/'+ldassa_tmp
-                        if (os.path.isfile(LDASsa_pertRstFile)) :
-                            self.has_ldassa_pert = True
-               
+
+            elif (int(self.rqdExeInp['RESTART']) == 0) : 
+              if (self.catch == 'catch'):
+                self.in_rstfile = '/discover/nobackup/projects/gmao/ssd/land/l_data/LandRestarts_for_Regridding' \
+                                  '/Catch/M09/20170101/catch_internal_rst'   
+                self.in_tilefile = '/discover/nobackup/projects/gmao/ssd/land/l_data/geos5/bcs/CLSM_params' \
+                                   '/mkCatchParam_SMAP_L4SM_v002/SMAP_EASEv2_M09/SMAP_EASEv2_M09_3856x1624.til'
+              elif (self.catch == 'catchcnclm40'):
+                self.in_rstfile  = '/discover/nobackup/projects/gmao/ssd/land/l_data/LandRestarts_for_Regridding' \
+                                  '/CatchCN/M09/20151231/catchcn_internal_rst' 
+                self.in_tilefile = '/discover/nobackup/ltakacs/bcs/Heracles-NL/SMAP_EASEv2_M09/SMAP_EASEv2_M09_3856x1624.til'
+              elif (self.catch == 'catchcnclm45'):
```

Comment:
@weiyuan-jiang, @gmao-jkolassa:
If I'm guessing right, the current ldas_setup does not have information about a dummy restart file for CatchCNCLM45.  It looks like the dummy restart files for the other model versions are all in the M09 tile space.  @gmao-jkolassa, do you have an M09 restart for CatchCNCLM45 that could be used as a dummy restart here?  If not, we could use the M36 restart from the LDAS_GLOBALCNCLM45 nightly test.
@weiyuan-jiang, please work with @gmao-jkolassa to implement a dummy CatchCNCLM45 restart in ldas_setup.

Reply:
I do not have an M09 restart file readily available, but I can generate one if that is preferred. Please let me know.

@weiyuan-jiang I will send you the path to an M36 restart that can be used if an M09 restart is not needed.

Reply:
@gmao-jkolassa: I'll leave it up to you to decide if we need an M09 dummy restart or if the M36 restart from the nightly test case is sufficient.  Note that this is just a dummy and meant to give users a start by providing a properly formatted restart.  Users will need to do some spin-up (unless the dummy restart itself is spun up properly, in the same tile space, and for the desired restart date, which is not likely to ever happen).

Reply:
Since this is only meant to serve as a dummy restart with the understanding that some additional spin-up is necessary, I think it is fine to use the M36 restart file from the nightly test. I will work with @weiyuan-jiang to make sure it is copied to an accessible space.